src/test.js

```diff
@@ -70,9 +70,11 @@ Test.prototype = {
 
 		config.current = this;
 
+		if (this.module.testEnvironment) {
```

Comment:
Minor style issue: Missing spaces inside `if`. Not sure why jscs didn't catch this.